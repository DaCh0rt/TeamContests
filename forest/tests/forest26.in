912 362
-8989 3350
-1273 3221
3713 2575
-5787 -7441
-101 -2284
-2950 8381
-5137 -4986
-2210 5781
1409 9311
1526 9683
4182 -1592
9322 9195
4059 -2069
7934 -212
741 8686
6903 8043
-2914 2633
-1489 8750
5058 5014
2808 -6033
-5842 -235
-7191 -6175
-6439 3672
-8099 -9680
-7000 8134
2597 3367
-54 -6212
8516 4744
5453 -8927
3599 1236
-6730 5217
4360 6069
4391 -3550
3728 9845
-2138 2491
3972 -6257
3699 -242
-1616 -5485
-6263 -1683
2511 -8615
7665 4967
9354 -6165
6400 -1903
-7309 -8837
4079 6937
9440 5804
-8931 -760
-4107 4890
-7770 86
-3389 -1511
-505 -7815
8331 -3258
5011 -9698
742 5036
5693 4293
3488 -3320
8335 -3751
9351 5223
9881 -4271
684 -5475
-5028 -7531
-2782 -1742
1195 3139
4797 907
-356 8039
9531 -777
-5835 2094
3005 -6183
5366 1076
4030 1891
-2958 -2788
-3157 -3483
9044 7052
6356 -947
-3613 5571
6587 6792
-2575 -2250
9644 6445
3480 8418
7789 -8328
5599 -2587
4489 8764
3035 6418
7273 1424
3152 7105
1079 7500
-6106 2738
-1492 -8833
-8869 2576
-8562 -9597
-1514 -6272
-1029 5574
-7215 -6501
5897 -9879
-3071 485
-9535 -1426
-929 -4002
-4841 5298
4252 384
9347 -318
3276 7989
-9868 -2403
1943 5720
-4342 5633
-747 5017
4208 -8912
2821 -1429
6689 8153
-3638 -7734
-9772 -3041
9627 25
6501 3197
-7483 3528
-2479 -5506
-4641 6301
9647 -7254
4194 841
5396 -4816
1342 -2366
6298 -5415
2719 -5009
8529 5552
3713 6394
924 5498
49 7490
-7371 -377
-2816 -1346
769 6525
-6749 -7855
-2617 -2628
-5962 -1425
6102 7174
-6612 -6002
3344 -1111
-3360 -3694
3137 -52
2546 4005
-5339 1231
9617 9254
-5921 765
1290 9916
-6395 -3580
7969 9892
3934 -7539
1479 -2614
7090 -6081
9972 3272
-3500 -9640
-3633 8245
5164 -4443
-2330 -6113
-269 -8086
8918 5832
-1860 4942
-6277 -133
901 8053
3350 9212
3448 1383
3834 4546
-8492 3077
9191 -4197
6945 -5958
-4587 -6915
6345 7016
-965 7347
1481 -8091
-3118 41
2969 -2688
6774 -3157
-1962 2968
5720 1056
-6653 -8831
3956 4380
8278 -5466
8700 6255
-7962 -7972
6392 -5079
-4036 2958
-9259 2509
-5818 8362
-3555 -6451
4189 985
-8034 -1463
8740 7810
72 -6245
-4538 -8106
-6661 8797
3067 1119
-4310 264
3131 -2034
7048 -490
-9904 7710
-7292 -501
9469 -1895
-1087 -717
-8692 -6785
-8708 4071
-3478 -4153
-6693 -8565
9247 9478
9756 8581
-61 -520
1126 9910
-5581 6302
4828 5932
-747 2760
2421 -1095
7364 2827
-8979 -9022
-4123 -4944
2571 -3819
-9704 -8161
-4730 -4664
-4237 8376
2984 -5504
4450 9604
4033 6818
-5312 -9992
-5312 -3845
-9521 -6475
-4666 -8961
-6836 -6065
-7838 5275
3591 -5218
-9415 7875
1085 4819
9903 9282
-7142 9676
-9917 -6793
-6385 806
5738 -8062
4709 1884
2956 1675
-3931 -2086
68 -1388
9784 5047
8750 3100
8818 1460
-2239 5313
6713 9977
5624 -5369
-3979 8612
-9128 1787
7094 6409
4135 -3315
3191 -4244
4643 3090
7389 9394
-4889 6336
2896 7050
7386 -3944
-7836 1817
-3828 3574
-2589 7539
7217 7967
6631 5632
-9773 3096
8695 -6196
3093 -6097
-6645 8121
-416 -2456
-9073 9070
-5843 5997
3027 6414
5622 -3927
5073 -1535
-5152 8573
-4104 9532
-8471 -9536
4623 2263
-2416 3147
5603 -8575
-9336 884
3204 8496
1611 3414
-1248 2370
-4434 -7804
5176 -7169
-3627 -7342
-3762 -3618
-5634 5303
2888 8259
-9821 2727
-7318 1578
2082 2104
-9802 9426
-1467 -4575
4539 1077
-3423 8717
-393 3683
126 760
7070 -4261
-1074 -9765
-6891 -5151
590 -6218
-306 -6771
-7303 6104
7967 796
7522 -1893
8437 -182
-868 6753
-9342 2851
-8541 6434
9647 -6330
-4636 3524
4812 -9624
-2634 2690
4963 2729
-9877 7186
3471 -4476
-4449 -4762
3139 -4189
3668 -476
2059 -7372
-291 6634
9531 7856
-3541 -2364
2732 1176
9668 -7065
-2468 2969
-1185 -2089
9198 7889
5600 2331
-85 3032
7108 -9317
-9922 -2351
3666 9053
9742 -9824
-7567 1707
2128 3975
6209 -7476
-3352 9014
9739 -688
-6985 -8799
-452 -3214
5386 -6648
-3553 7698
9457 7001
54 590
-1717 -9800
-5586 -4376
1584 -5917
1543 -8508
-5294 -351
-1193 7692
4819 -916
3292 4512
-5014 -158
-9562 9415
5549 -5005
795 -6296
-269 -4577
7970 -578
6000 7485
525 -2434
-1120 6885
-9406 8774
6652 2508
7157 5426
275 7980
-9389 -5742
5740 -1434
-4602 -7627
-2819 -4938
4332 -7753
9655 -5992
9488 -5112
8375 -1722
1654 3956
7602 -2167
-8609 -753
6255 2917
5378 8783
-4978 -3836
-7760 -4342
8060 -7596
-8301 -1423
4011 -9514
-5369 365
4534 -4322
2263 -3864
-6992 7084
-7765 -2031
5198 1008
-7813 -6546
-9623 -5963
2522 -856
-1230 717
-463 -8695
-9870 -2219
-2052 -597
8360 -9224
4312 -2537
4512 3
5241 3844
1348 9534
-6191 7211
-8045 -5657
7692 -1674
3713 788
7768 -1394
4455 -7708
8327 7888
6584 7574
5740 -1506
-307 8982
-3368 3056
7220 -7269
3226 -2036
7352 215
6442 4667
-4874 -458
6863 -9565
-7690 3973
9356 6206
-6912 2282
5075 -5288
-2533 -9435
264 -1879
-804 693
-304 -8372
8653 -6779
2649 -5557
-8513 -7923
24 -2033
-4892 -6586
-6485 -1689
-8713 3737
5437 4730
-5480 4431
-2518 2022
4048 -763
-6923 -5706
3812 8786
9330 1087
2858 -6200
-8746 -7689
3846 -1929
-9896 -7346
8758 1342
-4627 6292
-8702 -8668
-6456 -5946
-6649 2243
603 271
-8960 1263
8906 -8145
-3698 -3984
-5862 -466
4364 3508
7351 -4287
7143 2692
3120 -2338
-6665 2767
-4541 -409
-5540 1765
-3958 5264
9527 -7664
8645 4759
-4596 -3254
-4457 373
-9645 6909
-400 -1095
834 -4848
9770 9936
-746 6658
-7742 8254
-7181 2112
-1649 5579
-8799 -7898
8710 -5205
3627 -8374
6832 6471
-1708 -6183
-776 3823
-1940 -6273
6666 9216
4590 2337
-8814 6862
-5463 1643
8194 -3568
-5815 2096
-3405 -3987
-7716 -263
9572 8495
3246 -7907
-6409 7177
-1870 8218
2793 -2128
-9890 1208
-7209 8978
2125 7479
5591 3928
5003 -7339
4347 -1607
8431 -3706
2376 -4133
-3276 -1388
-625 1888
-8550 -8377
-5733 360
-5213 -4160
5299 7193
5219 3482
8544 -8792
-4879 -6889
8585 3540
6741 -3238
2509 -6245
877 -9888
-7520 5491
-3583 8898
9289 -5640
7104 149
-7701 -5124
1159 -2193
-3710 -9517
-7889 8321
-8727 6991
-2860 4436
-496 442
-6422 -7697
4294 6287
-5076 1188
-5591 3461
7294 3991
2272 -8122
2146 8641
-7139 -9142
-6778 -6541
151 -4213
8910 9980
4957 -9171
-3297 9497
-7104 -2895
-2425 4661
6533 -3106
8218 2755
-1877 7933
-4540 2172
7186 8872
5027 -7128
8795 2205
-2224 6082
-9299 3441
982 810
-1495 -8845
-651 -7427
393 -9033
-1755 3980
1201 -5971
1319 -2969
6613 9072
-1535 -212
-7421 -6394
2987 -1071
-7536 -4338
-7010 3478
-1761 -194
-2515 3322
9139 5043
5523 -7871
3200 -9978
-4107 5996
-6427 -9276
-7622 8606
2242 1509
2324 -2010
9545 7180
-6823 -6277
7680 -7018
-6558 -8707
2583 -2466
1977 252
8407 8181
-5725 -6979
-3748 5891
-6632 191
-9206 -4841
966 1203
2107 -4901
4248 2040
-8797 -3361
-7720 -5790
-8393 6147
-6410 -9538
2453 -3417
5313 -4091
-6495 7269
9969 7746
8759 7606
-7579 -7064
-2539 7263
-7375 -8788
519 4827
6074 -5660
-9435 -9867
-1416 9153
6286 3653
-1299 -1115
5529 -3371
9568 9379
7129 -7078
-8587 -6843
9250 -687
8868 -8743
1703 -2702
-871 9369
-9682 -9485
199 -5520
9172 -7715
-5468 -8552
1576 -6208
9251 -351
-9619 -9147
8671 -7156
5334 7043
-9411 -7798
-7456 1283
-7911 6809
1763 4662
7880 -7779
-6944 615
3525 -9297
-971 -3995
5172 -6757
7694 -4511
-3475 4222
8877 670
6529 -2024
4678 -9886
9667 2931
-4218 -420
-1137 -4835
7274 5208
-8994 6353
-5420 6275
5412 -269
-5363 6032
1384 -4947
-9685 -6501
5605 4674
7954 9767
1154 -3908
3298 6116
-9736 218
7003 1256
8128 -3980
-7078 -4208
3602 1103
3815 -7067
-5124 4622
6810 -2986
-8261 -6183
2590 7884
2096 1649
-7427 -6755
-6628 -6412
901 9014
2036 -1912
-7186 927
-9691 -437
4822 -3292
-2216 -1094
-3278 -5778
8870 -3848
4992 -7203
-3298 635
96 -802
-8586 3801
-4706 634
-1832 -4777
6970 -98
-5248 8329
-2589 -9993
-3326 -3142
-2443 -8021
-2562 -2639
-4924 -6139
-3646 2488
-7207 -8497
3560 -3330
7793 4640
2390 2358
-3267 4242
-8785 612
199 -5711
2725 8196
8491 7657
-941 6432
-7348 2476
9809 4661
2613 -6491
6441 -1194
5733 -7670
7004 7702
1481 -8778
9483 8249
-908 -341
2137 -4180
2038 767
7459 3349
-69 3383
-691 -9669
-4971 6065
3780 -5973
496 -5755
-3129 -129
-143 -6356
-3262 -4121
-686 -3765
1495 -1739
3508 -2798
-9408 -4970
-5315 4255
8192 -5827
1055 2396
1559 -5795
-7449 -8786
9999 -4946
-19 4645
4916 -715
2885 -1844
-3396 -3362
8047 -713
-107 -1890
-7466 3665
8569 9698
3238 7177
-8027 1996
-9494 -2582
-2388 -369
7988 -5145
-1646 -6342
-304 7290
-882 6728
-2534 9587
-4398 3792
8724 -7102
-5776 2134
-8213 -4355
-6292 -3130
8823 -9971
-6131 -9710
6864 -8262
1431 8449
-1126 -8864
8355 578
-6351 9705
-6585 -3434
-7584 -3285
-3157 2215
7731 7425
3152 9462
-1286 -5050
3155 4368
-1780 9730
-4453 -3186
9192 2564
5371 -2570
1933 -9613
-9289 -6838
-1992 -4826
-3137 3061
-3221 -7944
6027 -6376
-6279 -2545
6569 932
3869 4876
-4903 -9656
-9355 -6891
1591 1560
3202 -5665
5792 2928
3127 -3283
-1352 6425
-7028 -2431
-4592 4998
268 -1271
-3700 -1391
-3293 4197
7878 4975
-9457 4381
5433 -9571
4857 4996
5499 -5397
-6326 8202
4162 5209
182 -5212
-3415 -7865
7505 7407
-2358 2037
-1065 -1507
-3221 2230
2130 1586
-3215 8433
-1637 -9448
8899 390
6244 -4492
1194 1298
-255 1304
8615 -9031
-7985 3693
-7373 9816
-678 7204
-3449 -7559
-7829 -7128
-5913 2874
4317 6438
6564 9752
3379 9654
2382 6404
-307 -6227
-4194 1456
-3917 -3987
5246 2604
-1734 3347
8905 8920
-6646 -7798
-1019 6259
4806 -6501
1539 5884
-8514 7207
7188 7118
-1561 8688
-1233 8956
3451 5932
-1589 -9438
-9497 -3497
3499 344
9576 -7808
-4771 -2331
383 -8144
1993 4998
2176 -6455
-6601 7582
9081 7227
7353 -4725
-5075 7902
4180 7747
-7405 -9922
7287 6881
6698 -2677
2398 1771
-6990 607
-2357 -9986
-4870 -5979
-8945 3250
-1441 3582
2811 6975
-7663 -7958
6798 5531
-2416 -2244
9264 1690
-4564 2702
-4100 -2620
2228 -4650
-5279 8180
-4694 2266
8006 9231
-7030 -5394
2453 2436
8952 4971
-692 102
4173 -4936
6754 -7682
-5804 -1792
5034 -4424
5313 7459
4759 5872
9236 1963
-2664 7191
3754 7956
2871 3492
-4738 9518
4958 -6467
9884 1693
-9896 -8411
-8473 2886
-785 -5936
-3258 5810
1625 -5068
9480 3491
-566 -3014
-2017 8631
511 -2315
-1826 4005
-1383 7650
9722 8434
3742 5736
2844 -1845
-8059 -3559
5749 8661
9845 359
1938 2168
5588 -645
-1738 6157
-8265 6211
-9168 -7168
6523 5542
2485 -8876
-4036 -4653
-4038 808
-397 9183
-844 -4143
-3682 -3446
-8232 -2709
-9251 -5547
145 -9775
-7802 2579
5008 9082
-5582 -7846
