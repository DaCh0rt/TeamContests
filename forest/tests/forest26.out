80242.07 422249694.62
